box/box.lua

```diff
@@ -0,0 +1,78 @@
+--[[
+
+# [Box](https://developer.box.com/docs) connector.
+
+### Authorization
+
+Authorization use [Authentication with OAuth](https://app.box.com/developers/console).
```

Comment:
The link you provided requires authentication, but https://developer.box.com/docs/oauth-20 don't